fern/definition/apigateway.yml

```diff
@@ -0,0 +1,45 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/fern-api/fern/main/fern.schema.json
+types:
+  RestApi:
+    properties:
+      baseUrl: string
+      name: string
+      region: string
+      createdTime: datetime
+      stage: string
+      paths: list<RestApiPath>
```

Comment:
One of the things we've found is that the Fern go vs python don't serialize empty lists the same. If this could ever be empty, I suggest making this `optional<list<RestApiPath>>`